databases/week3/part2_mealapp/part2_meal_app_queries.sql

```diff
@@ -0,0 +1,114 @@
+use meal_application;
+
+-- 1.Get all meals
+select * from meal;
+
+-- 2.Add a new meal
+insert into meal (title,description,location,`when`,max_reservations,price,created_date) values ('pizza','margherita','sydhavn','2020-02-22 13:30:20','6','65','2020-02-21');
+
+-- 3.Get a meal with any id, fx 1
+select *
+from meal
+where id=1;
+
+-- 4.Update a meal with any id, fx 1. Update any attribute fx the title or multiple attributes
+update meal
+set title = 'KFC burger'
+where id=1;
+
+-- 5.Delete a meal with any id, fx 1
+delete from meal
+where id=2;
+
+
+
+
+
+-- 1.Get all reservations
+select * from reservation;
+
+-- 2.Add a new reservation
+insert into reservation (number_of_guests,meal_id,created_date) values ('1',6,'2020-02-21');
+
+-- 3.Get a reservation with any id, fx 1
+select *
+from reservation
+where id=1;
+
+-- 4.Update a reservation with any id, fx 1. Update any attribute fx the title or multiple attributes
+update reservation
+set number_of_guests = '2'
+where id=1;
+
+-- 5.Delete a reservation with any id, fx 1
+delete from reservation
+where id=2;
+
+
+
+
+
+-- 1.Get all reviews
+select * from review;
+
+-- 2.Add a new review
+insert into review (title,description,meal_id,stars) values ('fantastic','tasty food',6,'5');
+
+-- 3.Get a review with any id, fx 1
+select *
+from review
+where id=3;
+
+-- 4.Update a review with any id, fx 1. Update any attribute fx the title or multiple attributes
+update review
+set title = 'very nice'
+where id=4;
+
+-- 5.Delete a review with any id, fx 1
+delete from review
+where id=2;
+
+
+
+-- additional queries
+-- 1.Get meals that has a price smaller than a specific price fx 90
+select *
+from meal
+where price < '70';
+
+-- 2.Get meals that still has available reservations
+select meal.title, meal.description, meal.max_reservations, reservation.number_of_guests, meal.max_reservations-reservation.number_of_guests as available_reservations
+from meal inner join reservation on reservation.meal_id = meal.id
+where meal.max_reservations > reservation.number_of_guests;
+
```

Comment:
This one does not produce the answer
Please read my comment on the channel. Thank you

Reply:
Do you need help with this?

Reply:
implemented feedback